nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexSurefireExecution.java

```diff
@@ -91,6 +91,34 @@ protected void setupArgline() {
 
     }
 
+    @Override
+    protected Xpp3Dom applyNonDexConfig(Xpp3Dom configuration) {
+        return addExcludedGroups(super.applyNonDexConfig(configuration));
+    }
+
+    private Xpp3Dom addExcludedGroups(Xpp3Dom configNode) {
```

Comment:
That is not what was stopping their execution. I think my current fix should fix also their running in the clean. Can you test that? Let me know if my change fixes also the clean run.